site/app/Resources/views/base.html.twig

```diff
@@ -155,6 +150,12 @@
       <script src="{{ asset_url }}"></script>
     {% endjavascripts %}
 
+    {{ encore_entry_script_tags('app') }}
+
+    <!-- Renders app.js & a webpack runtime.js file -->
```

Comment:
Does this really render the files, or just include them?

Reply:
Includes the `build` files